activesupport/lib/active_support/cache.rb

```diff
@@ -63,7 +63,13 @@ def lookup_store(store = nil, *parameters)
         case store
         when Symbol
           options = parameters.extract_options!
-          retrieve_store_class(store).new(*parameters, **options)
+          # clean this up once Ruby 2.7 support is dropped
+          # see https://github.com/rails/rails/pull/41522#discussion_r581186602
+          if options.blank?
```

Comment:
The `options` is exactly a Hash object, just use the Ruby native method `options.empty?`.